components/brave_rewards/resources/extension/brave_rewards/adaptive_captcha_panel.html

```diff
@@ -0,0 +1,17 @@
+<!doctype html>
```

Comment:
seems weird to have this stuff in rewards since rewards does not depend on captcha. Seems like this stuff should go in the adaptive captcha component?